### DISCUSSION TITLE
Add Korean Language Support

### DISCUSSION THREAD

**Discussion by NullByte42** [ente-io/ente#4112] [Enhancements]
Hi Ente Team,

I love using Ente! However, I noticed that Korean language support is not yet available. Adding Korean would make the app more accessible for Korean-speaking users. Please consider adding this feature in future updates.

Thank you!

**Comment by ua741** [comment]
The Korean language is already listed in both of our apps. Once 90% of the strings are translated, we will enable support for it in the app.
https://crowdin.com/project/ente-authenticator-app
https://crowdin.com/project/ente-photos-app
